packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -20,6 +20,26 @@ export const sendBoxStyleSet = {
   }
 };
 
+/**
+ * @private
+ */
+export const sendBoxStyleMain = mergeStyles({
+  padding: '0rem',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flexStart',
```

Comment:
I advise using `Stack` instead of div since it is a flexbox and provides props for most of these alignment styles.

Reply:
done